Assets/GothicVR/Scripts/Creator/WorldCreator.cs

```diff
@@ -185,7 +189,7 @@ public void LoadEditorWorld(IntPtr vdfPtr, string zen)
             sampleScene.GetRootGameObjects().Append(worldGo);
 
             // load only the world mesh
-            SingletonBehaviour<MeshCreator>.GetOrCreate().Create(world, worldGo);
+            await SingletonBehaviour<MeshCreator>.GetOrCreate().Create(world, worldGo, progress => { });
```

Comment:
```suggestion
            await MeshCreator.I.Create(world, worldGo, progress => { });
```

Why are you putting a progress function into?

Reply:
empty action callback that will be removed when I do the LoadingManager stuff